common/viewer.cpp

```diff
@@ -2239,7 +2243,7 @@ namespace rs2
 
         glDisable(GL_DEPTH_TEST);
 
-        if (ImGui::IsKeyPressed('R') || ImGui::IsKeyPressed('r'))
+        if (ImGui::GetIO().KeysDown[ImGuiKey_R])
```

Comment:
Will catch both upper/lower case?

Reply:
yes, it only checks if the Z key is pressed (will work for uppercase and lowercase letters).